trussworks/react-uswds
# feat: Add Pagination component

# Summary
Adds a new Pagination component.

This is a port of the Pagination component that was built for the [USSF Portal](https://github.com/USSF-ORBIT/ussf-portal-client/tree/main/src/components/Pagination). I also wanted to note that I followed the instructions [here](https://github.com/trussworks/react-uswds/blob/main/docs/contributing.md#yarn-link) and used `yarn link` to successfully test the component in the USSF Portal.

## Related Issues or PRs
closes: #1543 


## How To Test
1. Run `yarn storybook` and look at `Components/Pagination/Sandbox` and change some of the controls to see how the component adjusts.
2. Run `yarn test:coverage` and verify that all tests are passing and that the coverage threshold is being met.




### Screenshots:
![pagination-1](https://user-images.githubusercontent.com/99674188/186696237-0d0ed868-cbb1-4dad-90f2-fc5de2330ab6.png)
![pagination-2](https://user-images.githubusercontent.com/99674188/186696266-9e4b1b97-c246-44a8-ba69-350619561c2c.png)
